airbyte-integrations/connectors/source-xkcd/metadata.yaml

```diff
@@ -1,13 +1,17 @@
 data:
+  allowedHosts:
+    hosts:
+      - xkcd.com
   connectorSubtype: api
   connectorType: source
   definitionId: 80fddd16-17bd-4c0c-bf4a-80df7863fc9d
-  dockerImageTag: 0.1.10
+  dockerImageTag: 0.2.0
   dockerRepository: airbyte/source-xkcd
```

Comment:
This is also missing the block with build options → base image. Look up what is the latest `airbyte/source-declarative-manifest:3.9.0` sha hash is, and put the whole block here. I can look tomorrow, but it's 10pm and my brain is fried today. Let's make this a manifest-only connector!

Reply:
Yes sure!